proposals/simd/SIMD.md

```diff
@@ -784,3 +784,25 @@ Lane-wise saturating conversion from floating point to integer using the IEEE
 resulting lane is 0. If the rounded integer value of a lane is outside the
 range of the destination type, the result is saturated to the nearest
 representable integer value.
+
+### Integer to integer widening
+* `i16x8.convert_s/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_s/i8x16High(a: v128) -> v128`
+* `i16x8.convert_u/i8x16Low(a: v128) -> v128`
+* `i16x8.convert_u/i8x16High(a: v128) -> v128`
+* `i32x4.convert_s/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_s/i16x8High(a: v128) -> v128`
+* `i32x4.convert_u/i16x8Low(a: v128) -> v128`
+* `i32x4.convert_u/i16x8High(a: v128) -> v128`
```

Comment:
These names should probably be updated since capital letters and slashes are now inconsistent with the rest of the instructions. How about `i16x8.{widen,narrow}_i8x16_s` and similar.

Reply:
The proposal still uses the old scheme, I'm okay with the proposed {widen, narrow} convention - will update the rest of the proposal to match too.

Reply:
what would i32x4.convert_s/i16x8Low look like?
- i32x4.widen_i16x8_s_low
- i32x4.widen_low_i16x8_s

Reply:
I think we agreed on a convention that the `_s` or `_u` signifying "signed" or "unsigned" is always at the very end of the instruction, so it would be your second example.

Reply:
Done, changed all the widen/narrow operation names to follow this convention.